src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -36,6 +36,38 @@ public static Vector3 GetOrganelleDirection(OrganelleTemplate organelle)
         return (Hex.AxialToCartesian(new Hex(0, 0)) - Hex.AxialToCartesian(organelle.Position)).Normalized();
     }
 
+    public static float GetNominalCapacityForOrganelle(OrganelleUpgrades? upgrades, OrganelleDefinition definition)
```

Comment:
```suggestion
    public static float GetNominalCapacityForOrganelle(OrganelleDefinition definition, OrganelleUpgrades? upgrades)
```
I'd prefer the method parameters to be this way around as the definition is the more important parameter, and also upgrades is allowed to be null.